ogr/services/forgejo/project.py

```diff
@@ -307,6 +307,7 @@ def remove_group(self, group: str) -> None:
         raise NotImplementedError()
 
     @indirect(ForgejoIssue.get_list)
+    # type: ignore
```

Comment:
```suggestion
    # type: ignore[empty-body]
```

Reply:
Same applies elsewhere. It's better to pass an explicit error code when to `type: ignore` pragmas, but maybe it doesn't matter so much here.